Solutions/CiscoSecureEndpoint/Analytic Rules/CiscoSEWebshell.yaml

```diff
@@ -0,0 +1,32 @@
+id: b6df3e11-de70-4779-ac9a-276c454a9025

```

Comment:
There should not be 2 templates with the same ID, but the id b6df3e11-de70-4779-ac9a-276c454a9025 is duplicated.